src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/administration.html.twig

```diff
@@ -93,28 +60,7 @@
           </h3>
           <div class="card-block row">
             <div class="card-text">
-              <div class="form-group row">
-                {{ ps.label_with_help(('Maximum size for attached files'|trans), ('Set the maximum size allowed for attachment files (in megabytes). This value has to be lower or equal to the maximum file upload allotted by your server (currently: %size% MB).'|trans({'%size%': 'PS_ATTACHMENT_MAXIMUM_SIZE'|configuration}, 'Admin.Advparameters.Help'))) }}
-                <div class="col-sm">
-                  {{ form_errors(uploadQuotaForm.max_size_attached_files) }}
-                  {{ form_widget(uploadQuotaForm.max_size_attached_files) }}
-                </div>
-              </div>
-              <div class="form-group row">
-                {{ ps.label_with_help(('Maximum size for a downloadable product'|trans), ('Define the upload limit for a downloadable product (in megabytes). This value has to be lower or equal to the maximum file upload allotted by your server (currently: %size% MB).'|trans({'%size%': 'PS_LIMIT_UPLOAD_FILE_VALUE'|configuration}, 'Admin.Advparameters.Help'))) }}
-                <div class="col-sm">
-                  {{ form_errors(uploadQuotaForm.max_size_downloadable_product) }}
-                  {{ form_widget(uploadQuotaForm.max_size_downloadable_product) }}
-                </div>
-              </div>
-              <div class="form-group row">
-                {{ ps.label_with_help(("Maximum size for a product's image"|trans), ('Define the upload limit for an image (in megabytes). This value has to be lower or equal to the maximum file upload allotted by your server (currently: %size% MB).'|trans({'%size%': 'PS_LIMIT_UPLOAD_IMAGE_VALUE'|configuration}, 'Admin.Advparameters.Help'))) }}
-                <div class="col-sm">
-                  {{ form_errors(uploadQuotaForm.max_size_product_image) }}
-                  {{ form_widget(uploadQuotaForm.max_size_product_image) }}
-                </div>
-              </div>
-              {{ form_rest(uploadQuotaForm) }}
+              {{ form_widget(uploadQuotaForm) }}
```

Comment:
This is the part I like in this PR ❤️

Reply:
Yeah removing code is one of the more satisfying parts of programming :D